### DISCUSSION TITLE
No tengo acceso a las sesiones

### DISCUSSION THREAD

**Discussion by gersondlarosa** [Launch-X-Latam/MisionFrontEnd#525] [Team Posts]
Buena noche aun no tengo el correo para la misión backen me podaran indicar donde la puedo obtener

**Comment by UlisesNh** [comment]
Dale click a este enlace para unirte bro
https://teams.microsoft.com/dl/launcher/launcher.html?url=%2F_%23%2Fl%2Fteam%2F19%3A59Tb8L_0PlM5uxQtC8AjVfcF-wvYCKBxsS1JRWZLnyo1%40thread.tacv2%2Fconversations%3FgroupId%3De6804a94-ce40-41ca-a5c2-b287fc50036b%26tenantId%3D4ae54b05-b77e-4224-aef1-8661422e0816&type=team&deeplinkId=3c6806b7-685d-4d93-9ffc-906dc320f8e3&directDl=true&msLaunch=true&enableMobilePage=true&suppressPrompt=true
